src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/GetAzureRMRecoveryServicesBackupProtectableItem.cs

```diff
@@ -80,10 +80,7 @@ public override void ExecuteCmdlet()
                     string containerName = "";
                     Dictionary<UriEnums, string> keyValueDict = HelperUtils.ParseUri(ParentID);
                     containerName = HelperUtils.GetContainerUri(keyValueDict, ParentID);
-                    if (containerName.Split(new string[] { ";" }, System.StringSplitOptions.None)[0].ToLower() == "vmappcontainer")
-                    {
-                        backupManagementType = ServiceClientModel.BackupManagementType.AzureWorkload;
-                    }
+                    backupManagementType = ServiceClientModel.BackupManagementType.AzureWorkload;
```

Comment:
@siddharth7 why was this change made? Was the command previously filtering out the wrong elements?

Reply:
We saw that we don't need this check. BackupManagementType will always be AzureWorkload for all upcoming ContainerTypes